packages/discovery-provider/src/tasks/index_aggregate_tips.py

```diff
@@ -194,15 +192,15 @@ def index_rank_ups(
 
 
 def _update_aggregate_tips(session: Session, redis: Redis):
-    latest_user_bank_slot = redis.get(latest_sol_user_bank_slot_key)
     prev_slot = get_last_indexed_checkpoint(session, AGGREGATE_TIPS)
-    max_slot_result = session.query(func.max(UserTip.slot)).one()
-    max_slot = int(max_slot_result[0]) if max_slot_result[0] is not None else 0
-
-    if prev_slot == max_slot:
-        if latest_user_bank_slot is not None:
-            redis.set(latest_sol_aggregate_tips_slot_key, int(latest_user_bank_slot))
-        return
+    max_slot_result = (
+        session.query(UserTip.slot, UserTip.signature)
+        .order_by(UserTip.slot.desc())
+        .one()
+    )
+    (max_slot, last_tip_signature) = (
```

Comment:
nit: i asked chatgpt and it said we could avoid the outer unpack:
`max_slot, last_tip_signature = max_slot_result if max_slot_result[0] is not None else (0, None)`

Reply:
wdym outer unpack? these look identical to me save for the parens (which I think is autoformatted? could be wrong)

Reply:
yeah it was the outer parens. nbd